### PR TITLE
Adding a link to the O/R Designer overview

### PR DESCRIPTION
**Pull request by fruby**
Currently the git repositories don't have any information about O/R designer. In order to save time from searches, this change adds a link to the msdn O/R designer overview.

# Title

On the title describe
what you've fixed (or created) with this Pull Request (PR).

## Summary

Insert a short (one or two sentence) summary here.

Fixes #Issue_Number

>Note: The "Fixes #nnn" syntax in the PR description causes
>GitHub to automatically close the issue when this PR is merged.
> Remove that line if you don't have issues associated with this
> PR. Click on the Guidelines for Contributing link above for details.

## Details

Explain your changes, and why you made them. If that
information is already available in the issue referenced
above, just referencing the issue is preferred to copying
the text.

This may not be necessary depending on the scope of the PR 
changes. For example, "fix typo in introduction.md" is
sufficient to describe that PR.

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

File: docs/framework/data/adonet/sql/linq/how-to-generate-the-object-model-in-visual-basic-or-csharp.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ manager: "jhubbard"
...
   
--   If you are using [!INCLUDE[vs_current_short](../../../../../../includes/vs-current-short-md.md)], you can use the [!INCLUDE[vs_ordesigner_long](../../../../../../includes/vs-ordesigner-long-md.md)] to generate an object model. The [!INCLUDE[vs_ordesigner_short](../../../../../../includes/vs-ordesigner-short-md.md)] provides a rich user interface to help you generate a [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] object model.  
+-   If you are using [!INCLUDE[vs_current_short](../../../../../../includes/vs-current-short-md.md)], you can use the [!INCLUDE[vs_ordesigner_long](../../../../../../includes/vs-ordesigner-long-md.md)] to generate an object model. The [!INCLUDE[vs_ordesigner_short](../../../../../../includes/vs-ordesigner-short-md.md)] provides a rich user interface to help you generate a [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] object model. For more information see the [Object Relational Designer](https://msdn.microsoft.com/en-us/library/bb384511.aspx) overview.
```
Comment (rpetrusha) [inline docs/framework/data/adonet/sql/linq/how-to-generate-the-object-model-in-visual-basic-or-csharp.md]: This is, unfortunately, not a good URL to use; this content is no longer being maintained. The current version of the article is available at https://docs.microsoft.com/en-us/visualstudio/data-tools/linq-to-sql-tools-in-visual-studio2. Could you modify the URL?

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @fruby, for contributing to the dotnet/docs repo and adding a cross-reference to O/R Designer. However, while the topic on MSDN is no longer being maintained, it does have an equivalent topic on docs.microsoft.com that is being maintained. Would you mind changing the URL to point to it? If you'd prefer to, we could make the change for you instead.

File: docs/framework/data/adonet/sql/linq/how-to-generate-the-object-model-in-visual-basic-or-csharp.md (hunk 1 above)
Comment (fruby) [reply docs/framework/data/adonet/sql/linq/how-to-generate-the-object-model-in-visual-basic-or-csharp.md]: Certainly! Thank you for the feedback

**Review by fruby — COMMENTED** [review]
Addressing pull request feedback

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thank you so much, @fruby, for making the additional change. We'll merge your PR once the build completes successfully. Your change should appear on docs.microsoft.com within the next day or two.

**Pull request merged**